src/main/java/com/duckblade/osrs/dpscalc/calc/gearbonus/ObsidianGearBonus.java

```diff
@@ -0,0 +1,79 @@
+package com.duckblade.osrs.dpscalc.calc.gearbonus;
+
+import com.duckblade.osrs.dpscalc.calc.EquipmentItemIdsComputable;
+import com.duckblade.osrs.dpscalc.calc.compute.ComputeContext;
+import com.duckblade.osrs.dpscalc.calc.compute.ComputeInputs;
+import com.duckblade.osrs.dpscalc.calc.model.GearBonuses;
+import com.google.common.collect.ImmutableSet;
+import lombok.RequiredArgsConstructor;
+import net.runelite.api.EquipmentInventorySlot;
+import net.runelite.api.ItemID;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.Map;
+import java.util.Set;
+
+@Singleton
+@RequiredArgsConstructor(onConstructor = @__(@Inject))
+public class ObsidianGearBonus implements GearBonusComputable
+{
+	private static final Set<Integer> OBSIDIAN_WEAPONS = ImmutableSet.of(
+		// Is the staff affected? Wiki is unclear. Will anybody ever use it? We'll find out.
+		// ItemID.TOKTZMEJTAL,
+	ItemID.TOKTZXILAK,
+	ItemID.TOKTZXILEK,
+	ItemID.TZHAARKETEM,
+	ItemID.TZHAARKETOM,
+	ItemID.TZHAARKETOM_T
+	);
+
+	private static final Set<Integer> BERSERKER_NECKLACES = ImmutableSet.of(
+		ItemID.BERSERKER_NECKLACE,
+		ItemID.BERSERKER_NECKLACE_OR
+	);
+
+	private static final Set<Integer> OBSIDIAN_HELMETS = ImmutableSet.of(
+		ItemID.OBSIDIAN_HELMET
+	);
+
+	private static final Set<Integer> OBSIDIAN_PLATEBODIES = ImmutableSet.of(
+		ItemID.OBSIDIAN_PLATEBODY
+	);
+
+	private static final Set<Integer> OBSIDIAN_PLATELEGS = ImmutableSet.of(
+		ItemID.OBSIDIAN_PLATELEGS
+	);
+
+	private final EquipmentItemIdsComputable equipmentItemIdsComputable;
+
+	@Override
+	public boolean isApplicable(ComputeContext context)
+	{
+		// Obsidian armour bonuses only apply when attacking i.e. manual casting is forbidden
+		boolean castingSpell = context.get(ComputeInputs.ATTACK_STYLE).isManualCast();
```

Comment:
You have the staff commented out so this should just be attackStyle is not magic. Can change later if needed

Reply:
Can you clarify why it is preferable to check that attack style is not magic over manual casting? I don't understand why it's better.

Reply:
It would change from 

`context.get(ComputeInputs.ATTACK_STYLE).isManualCast();`

to 
`context.get(ComputeInputs.ATTACK_STYLE).getAttackType() == AttackType.MAGIC;`

Reply:
All manual cast is magic, but not all magic is manual cast. It guards against more bad data cases. The API half is designed to be agnostic to the input restraints of the UI half.

Reply:
Right, I see.